File: examples/python/create_group.py
import os
import sys

from kalourmade_arkesel import ArkeselClient

group_name = sys.argv[1] if len(sys.argv) > 1 else sys.exit("Usage: python create_group.py <group_name>")
client = ArkeselClient(os.environ["ARKESEL_API_KEY"])

client.groups.create(group_name)
client.groups.add_contacts(group_name, [
    {"phone_number": "233544919953", "first_name": "Arkesel", "last_name": "Dev"},
])

print(f"Created group '{group_name}' with 1 contact")

File: examples/node/createGroup.ts
import { ArkeselClient } from '@kalourmade/arkesel-sms';

const groupName = process.argv[2];
if (!groupName) {
  console.error('Usage: tsx createGroup.ts <group_name>');
  process.exit(1);
}

const client = new ArkeselClient(process.env.ARKESEL_API_KEY!);

await client.groups.create(groupName);
await client.groups.addContacts(groupName, [
  { phoneNumber: '233544919953', firstName: 'Arkesel', lastName: 'Dev' },
]);

console.log(`Created group '${groupName}' with 1 contact`);
